pkg/services/sqlstore/migrator/postgres_dialect.go

```diff
@@ -157,7 +157,7 @@ func (db *PostgresDialect) TruncateDBTables() error {
 			if _, err := sess.Exec(fmt.Sprintf("DELETE FROM %v WHERE dashboard_id != -1 AND org_id != -1;", db.Quote(table.Name))); err != nil {
 				return errutil.Wrapf(err, "failed to truncate table %q", table.Name)
 			}
-			if _, err := sess.Exec(fmt.Sprintf("ALTER SEQUENCE %v RESTART WITH 3;", db.Quote(fmt.Sprintf("%v_id_seq", table.Name)))); err != nil {
+			if _, err := sess.Exec(fmt.Sprintf("ALTER SEQUENCE %v RESTART WITH 5;", db.Quote(fmt.Sprintf("%v_id_seq", table.Name)))); err != nil {
```

Comment:
Can anyone think of a reason not to just remove this completely? Does it really matter what the IDs of new records are under test? We don't have any tests that rely on hard-coded IDs any more (I took them out a few months ago).

This is not 5, not 3, as the new behaviour is:
- migration creates rules with IDs 1,2
- new migration removes these rules, but auto_increment stays at 3
- new sqlstore logic creates new rules again, with IDs 3,4

Reply:
I'm afraid the order is important, but not absolutely sure, meaning that the default ones needs to always have the lowest id's. Seems like queries using `dashboard_acl` often are ordered by `id`. But doesn't need to mean anything more than we always sort permissions in the order they was created. Unfortunately I don't remember the details around this. @torkelo or @daniellee maybe remember. If they don't we might need to dig in a little deeper to see exactly what queries being made using dashboard_acl.

Reply:
> But doesn't need to mean anything more than we always sort permissions in the order they was created.

If we do have some ordering assumption, I don't think this will solve it unfortunately. If we apply this migration to a Grafana instance where additional acl rules have been created, the new default acl rules will have been created with a later timestamp.

I guess we could sort by `org_id` or something to force these defaults to the 'top' of the list, if we do indeed rely on this ordering.

Reply:
Yeah need to figure out if there are order assumptions...

Reply:
I've checked the code now, especially the code below that check whether a user has certain permission and I cannot see that order makes any difference:
https://github.com/grafana/grafana/blob/9d8f61c7388f4b50e6c1844887833037317de946/pkg/services/guardian/guardian.go#L115-L120

So yeah think we can remove above completely as you originally suggested.